login_page.html

```diff
@@ -110,6 +115,63 @@
 				<br><br>
 			</form>
 		</div>
+
+
+		<!-- Modal -->
+		<div id="create_account_modal" class="modal fade" role="dialog">
```

Comment:
Why is the create account thing a modal instead of a page? I was thinking it would be cleaner to have an entirely separate page for this, but maybe you're thinking about something I'm not?